keyvi/include/keyvi/dictionary/fsa/internal/bit_vector.h

```diff
@@ -76,20 +76,22 @@ struct BitVector final {
    */
   template <std::size_t TsizeOther>
   inline void SetVector(const BitVector<TsizeOther>& other, const size_t start_bit) {
-    size_t bytePosition = start_bit >> 5;
-    size_t bitPosition = start_bit & 31;
-    size_t write_length = std::min(bits_.size() - bytePosition, other.bits_.size());
+    size_t byte_position = start_bit >> 5;
```

Comment:
As we are touching this code, will be good to make variables `const` along the way.